tests/test_cmdline.py

```diff
@@ -0,0 +1,31 @@
+import zipline.__main__ as main
+from zipline.testing import ZiplineTestCase
+from zipline.testing.predicates import (
+    assert_equal,
+    assert_raises_str,
+)
+
+
+class CmdLineTestCase(ZiplineTestCase):
+
+    def init_instance_fixtures(self):
+        super(CmdLineTestCase, self).init_instance_fixtures()
+
+        main.extension_args = {}
```

Comment:
We shouldn't permanently mutate mutate this. Instead, we can use a patch context manager: https://docs.python.org/3/library/unittest.mock.html.

Note: because we are using a python2 compatibility layer, `mock` is not a submodule of `unittest`, it is a top-level package.